### ISSUE TITLE
Training speed not stable on K80

### ISSUE THREAD

**Issue by weixsong** [ibab/tensorflow-wavenet#324]
Hi, 

The training speed variance a lot on my K80:

step 19 - loss = 5.071, (4.180 sec/step)
step 20 - loss = 4.678, (4.107 sec/step)
step 21 - loss = 4.617, (56.564 sec/step)
step 22 - loss = 3.657, (4.073 sec/step)
step 23 - loss = 4.641, (4.986 sec/step)
step 24 - loss = 4.580, (54.894 sec/step)
step 25 - loss = 3.895, (4.017 sec/step)
step 26 - loss = 0.537, (11.104 sec/step)
step 27 - loss = 3.834, (4.113 sec/step)

Not sure why this happens.
Using default configuration, only given input wave data folder path.

So each mini-batch is same length, 100000 samples (Actually I changed to 40000 samples).

Is this related to K80 hardware?
